src/index.js

```diff
@@ -27,7 +27,7 @@ module.exports = (web3, options = {}) => {
     registryAddress: options.ensRegistryAddress
   }
 
-  const readFileFromApplication = (contentURI, path) => {
+  const getProviderFromURI = (contentURI, path, cb) => {

```

Comment:
What is the `cb`?

Reply:
oops, it’s me trying different styles and forgetting to remove a parameter

Reply:
Ah, cool!